cmd/docs/templates/404.tmpl

```diff
@@ -3,5 +3,9 @@
   <p class="mt-3">
     Document not found
   </p>
+  <p class="mt-3">
+    This is the documenation for the latest multi-cloud version of Convox. 
```

Comment:
Typo in documentation

Reply:
Probably say "You are viewing" instead of "This is" as the pronoun can be ambiguous.